README.rst

```diff
@@ -833,8 +833,9 @@ your loopback and RFC1918 IP addresses are blacklisted.
 
 This also requires the optional lxml and netaddr python dependencies to be
 installed.  This in turn requires the libxml2 library to be available - on
-Debian/Ubuntu this means ``apt-get install libxml2-dev``, or equivalent for
-your OS.
+Debian/Ubuntu this means ``apt-get install libxml2-dev && pip install lxml``,
+or equivalent for your OS (if you use PostgreSQL, you also need 
```

Comment:
surely you need psycopg2 whether or not you use URL previews. this doesn't belong in this section.